SASS compiler cannot resolve node module

Here's the error message I get when opening a .scss file:

```
Compile Error. 
See error list for details
module.js:340
    throw err;
          ^
Error: Cannot find module 'C:\USERS\MADSK\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\12.0EXP\EXTENSIONS\MADS KRISTENSEN\WEB ESSENTIALS 2013\1.6.2\Resources\nodejs\tools\node_modules\node-sass\bin\node-sass'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
```

--- a/EditorExtensions/WebEssentials2013.csproj
+++ b/EditorExtensions/WebEssentials2013.csproj
@@ -983,7 +983,7 @@
       <Content Include="Resources\nodejs\tools\**" Exclude="&#xD;&#xA;               **\node-sass\**; **\less\**; &#xD;&#xA;               **\mocha\**; **\jade\**; &#xD;&#xA;               **\*.md; **\*.markdown; **\*.html; **\*.txt; **\LICENSE; **\README; **\CHANGELOG; **\CNAME; **\*.old; **\*.patch; **\*.ico; &#xD;&#xA;               **\*.json; **\.*; **\Makefile.*; **\Rakefile; **\*.yml; &#xD;&#xA;               **\test.*; **\generate-*; &#xD;&#xA;               **\*.min.js; **\*-min.js; &#xD;&#xA;               **\media\**; **\images\**; **\man\**; &#xD;&#xA;               **\.bin\**; **\benchmark\**; **\build\**; **\scripts\**; &#xD;&#xA;               **\vendor\**; &#xD;&#xA;               **\test\**; **\tst\**; **\tests\**; **\testing\**; &#xD;&#xA;               **\examples\**; **\example\**; **\tools\**\tools\**">
         <IncludeInVSIX>true</IncludeInVSIX>
       </Content>
-      <Content Include="Resources\nodejs\tools\node_modules\node-sass\sass.js; Resources\nodejs\tools\node_modules\node-sass\lib\**; Resources\nodejs\tools\node_modules\node-sass\bin\win32-ia32\**">
+      <Content Include="Resources\nodejs\tools\node_modules\node-sass\sass.js; Resources\nodejs\tools\node_modules\node-sass\bin\node-sass; Resources\nodejs\tools\node_modules\node-sass\lib\**; Resources\nodejs\tools\node_modules\node-sass\bin\win32-ia32-*\**">
         <IncludeInVSIX>true</IncludeInVSIX>
       </Content>
       <Content Include="Resources\nodejs\tools\node_modules\less\lib\**; Resources\nodejs\tools\node_modules\less\bin\**">

--- a/WebEssentialsTests/Tests/Sass/SassCompilationTests.cs
+++ b/WebEssentialsTests/Tests/Sass/SassCompilationTests.cs
@@ -1,0 +1,37 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MadsKristensen.EditorExtensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace WebEssentialsTests
+{
+    [TestClass]
+    public class SassCompilationTests
+    {
+        [ClassInitialize]
+        public static void Initialize(TestContext c) { NodeExecutorBase.InUnitTests = true; }
+
+        private static readonly string BaseDirectory = Path.GetDirectoryName(typeof(SassCompilationTests).Assembly.Location);
+
+        [TestMethod]
+        public async Task SassBasicCompilationTest()
+        {
+            foreach (var sourceFile in Directory.EnumerateFiles(Path.Combine(BaseDirectory, "fixtures", "sass"), "*.scss", SearchOption.AllDirectories))
+            {
+                var compiledFile = Path.ChangeExtension(sourceFile, ".css");
+
+                if (!File.Exists(compiledFile))
+                    continue;
+
+                var compiled = await new SassCompiler().CompileString(File.ReadAllText(sourceFile), ".scss", ".css");
+                var expected = File.ReadAllText(compiledFile)
+                                   .Replace("\r", "");
+
+                compiled.Should().Be(expected);
+            }
+        }
+    }
+}

--- a/WebEssentialsTests/WebEssentialsTests.csproj
+++ b/WebEssentialsTests/WebEssentialsTests.csproj
@@ -110,6 +110,7 @@
     <Compile Include="Tests\Markdown\MarkdownOutlinerTests.cs" />
     <Compile Include="Tests\Markdown\MarkdownCodeBlockTests.cs" />
     <Compile Include="Tests\Markdown\TabAwareCharacterStreamTests.cs" />
+    <Compile Include="Tests\Sass\SassCompilationTests.cs" />
     <Compile Include="Tests\Shared\MockSnapshot.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Tests\NodeModules\NodeModuleImportedTests.cs" />
@@ -180,6 +181,8 @@
     <None Include="fixtures\module-resolution\packages\main-index\package.json" />
     <None Include="fixtures\module-resolution\packages\main\package.json" />
     <None Include="fixtures\coffee\source\test.coffee" />
+    <Content Include="fixtures\sass\extend.css" />
+    <None Include="fixtures\sass\extend.scss" />
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup />

--- a/WebEssentialsTests/fixtures/sass/extend.css
+++ b/WebEssentialsTests/fixtures/sass/extend.css
@@ -1,0 +1,13 @@
+﻿.message, .success, .error, .warning {
+  border: 1px solid #cccccc;
+  padding: 10px;
+  color: #333333; }
+
+.success {
+  border-color: green; }
+
+.error {
+  border-color: red; }
+
+.warning {
+  border-color: yellow; }

--- a/WebEssentialsTests/fixtures/sass/extend.scss
+++ b/WebEssentialsTests/fixtures/sass/extend.scss
@@ -1,0 +1,20 @@
+﻿.message {
+  border: 1px solid #ccc;
+  padding: 10px;
+  color: #333;
+}
+
+.success {
+  @extend .message;
+  border-color: green;
+}
+
+.error {
+  @extend .message;
+  border-color: red;
+}
+
+.warning {
+  @extend .message;
+  border-color: yellow;
+}

